kornia/color/colormap.py

```diff
@@ -1,145 +1,182 @@
-from abc import ABC
-from typing import List, Optional
+from __future__ import annotations
+
+from enum import Enum
+from typing import Optional, Union
 
 import torch
 from torch.nn.functional import interpolate
 
+import kornia.color._colormap_data as cm_data
+from kornia.color._colormap_data import RGBColor
 from kornia.core import Module, Tensor, tensor
 from kornia.core.check import KORNIA_CHECK_IS_GRAY
+from kornia.utils.helpers import deprecated
 
-RGBColor = List[float]
 
+class CMAP(Enum):
```

Comment:
just for consistency what do you think about renaming this to `ColorMapType` and to enforce explicit names